src/sentry/static/sentry/app/components/modals/sentryAppDetailsModal.tsx

```diff
@@ -62,6 +64,34 @@ export default class SentryAppDetailsModal extends AsyncComponent<Props, State>
     closeOnInstall: true,
   };
 
+  componentDidUpdate(prevProps: Props) {
+    //if the user changes org, count this as a fresh event to track
+    if (this.props.organization.id !== prevProps.organization.id) {
+      this.trackOpened();
+    }
+  }
+
+  componentDidMount() {
+    this.trackOpened();
+  }
+
+  trackOpened() {
+    const {sentryApp, organization, isInstalled} = this.props;
+    recordInteraction(sentryApp.slug, 'sentry_app_viewed');
+
+    trackIntegrationEvent(
+      {
+        eventKey: 'integrations.install_modal_opened',
+        eventName: 'Integrations: Install Modal Opened',
+        integration_type: 'sentry_app',
+        integration: sentryApp.slug,
+        already_installed: isInstalled,
+        view: 'external_install',
```

Comment:
how come this is `external_install` ?

Reply:
@MeredithAnya Good catch, this is wrong :)